doc/COMPILING/COMPILING-CMAKE-VCPKG.md

```diff
@@ -0,0 +1,75 @@
+# Disclaimer
+
+**WARNING**: CMake build is **NOT** officially supported and should be used for *dev purposes ONLY*.
+
+For the official way to build CataclysmDDA, see:
+  * [COMPILING.md](COMPILING.md)
+
+# Contents
+
+1. Prerequisites
+2. Configure
+3. Build
+4. Install
+5. Run
+
+# 1 Prerequisites
+
+`cmake` >= 3.20.0<br/>
+`vcpkg` from [vcpkg.io](https://vcpkg.io/en/getting-started.html)
+
+
+# 2 Configure
+
+## Presets
+The file `CMakePresets.json` contains all the presets.<br/>
+They will all build the code into the directory `out/build/<preset>/`.
+
+## Visual Studio
+The Standard toolbar shows the presets in the _Configuration_ drop-down box.<br/>
+From the main menu, select _Project -> Configure Cache_
+
+## Terminal
+Run the command
+ * `cmake --list-presets`<br/>
+It will show the presets available to you.
+The list changes based on the environment you are in.
+
+Run the command
+ * `cmake --preset <preset>`
+
+If different from `C:\vcpkg` you can
+  * append `-DVCPKG_ROOT=your\path` to the above command
+  * set your path via `VCPKG_ROOT` environment variable
+  * edit `VCPKG_ROOT` cache variable in `CMakePresets.json`
+
+# 3 Build
```

Comment:
The vcpkg part should to be first because the later steps depend on it being correct. Even though it's all in the configure section, they should still be correctly ordered because it's reasonable to expect people to execute the steps in order.
```suggestion
# 2 Configure

## Presets
The file `CMakePresets.json` contains all the presets.<br/>
They will all build the code into the directory `out/build/<preset>/`.

## vcpkg

If vcpkg is checked out in a location different from `C:\vcpkg`, eg. `C:\dev\vcpkg`, you must do one of the following:
  * append `-DVCPKG_ROOT=C:\dev\vcpkg` (or whatever the path is) to any cmake configure commands
  * set the environment variable `VCPKG_ROOT` to the path to the vcpkg checkout.
  * edit the `VCPKG_ROOT` cache variable in `CMakePresets.json` to the appropriate path

## Visual Studio
The Standard toolbar shows the presets in the _Configuration_ drop-down box.<br/>
From the main menu, select _Project -> Configure Cache_

## Terminal
Run the command
 * `cmake --list-presets`<br/>
It will show the presets available to you.
The list changes based on the environment you are in.

Run the command
 * `cmake --preset <preset>`
 
# 3 Build
```